gradle/shared-with-buildSrc/repositories.gradle.kts

```diff
@@ -16,23 +16,19 @@
 
 
 val repositoryMirrors = findMirrorUrls()
-rootProject.extensions.extraProperties.set("repositoryMirrors", repositoryMirrors)
 
-val gradleMirrorUrl = repositoryMirrors["gradle"]
-if (gradleMirrorUrl != null) {
-    project.allprojects {
-        buildscript.repositories {
-            maven {
-                name = "gradle-mirror"
-                url = uri(gradleMirrorUrl)
-            }
-        }
-        repositories {
-            maven {
-                name = "gradle-mirror"
-                url = uri(gradleMirrorUrl)
-            }
-        }
+project.allprojects {
+    buildscript.repositories {
```

Comment:
Can we do that/do we need to do this in `settings.gradle` via `pluginManagement.repositories`?

Reply:
I think `buildscript.repositories` and `pluginManagement.repositories` are different `RepositoryHandler`. However, seems like I didn't see why we need `buildscript.repositores` here. I'm going to remove it and see if any CI build complains.